docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2622,6 +2622,13 @@ function transferHostToMember(transferButton) {
   }
 }
 
+function toggleBreakout() {
+  var enableBox = document.getElementById("enable-breakout");
+  const meeting = getCurrentMeeting();
+  var enableStates = enableBox.checked == true ? true : false;
```

Comment:
nitpick: enableStates seems a bit weird name (why plural?), maybe just "enabledState"? Also the ternary operator seems redundant, you could just do `enabledState = enableBox.checked` and even the enableStates variable seems a bit redundant too, so we could just call `meeting.breakouts.toggleBreakout(enableBox.checked)`

Reply:
got it, thanks